lib/eyaml.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require "thor"
+require "rbnacl"
+require "base64"
+require "yaml"
+require "json"
+require "pathname"
+
+require "pry"
+
+class EYAML
+  class Error < StandardError; end
+  DEFAULT_KEYDIR = ENV["EJSON_KEYDIR"] || "/opt/ejson/keys"
```

Comment:
a constant should be constant (so it should only be `/opt/ejson/keys`). Maybe add a class method that does this `||`?